Migration_DB_Scripts/loop_estbid_users_fn.sql

```diff
@@ -0,0 +1,35 @@
+-- FUNCTION: migration.loop_estbid_users(integer)
+
+-- DROP FUNCTION migration.loop_estbid_users(integer);
+
+CREATE OR REPLACE FUNCTION migration.loop_estbid_users(
+		n integer DEFAULT 100)
+	    RETURNS void
+	    LANGUAGE 'plpgsql'
+
+	    COST 100
+	    VOLATILE 
+	AS $BODY$
+	DECLARE
+	        Allestbid  REFCURSOR;
+		        currentestbid record;
+			Begin 
+				OPEN Allestbid  for 
+				select establishment_id from establishment_user where establishment_id not in  (156182, 59, 248, 669, 187078, 215842, 162286, 2533, 2952, 200560, 225586, 3278, 60682, 5228, 12937, 232842, 10121, 10757, 216264, 12041, 17047, 177958, 136485,
```

Comment:
And why `public.establishment.id` here, when `ublic.establishment.nmdsid ` for establishments? Surely, you want to work with the same input data set?

Reply:
No I am excluding the the data set excluding the id's that's already mapped in.

I have not changed any functionality of your function, however using same approach I have excluded already migrated Id's so that it passes or accept the establishment id that is not part of already migrated record.

Thanks
Shakir